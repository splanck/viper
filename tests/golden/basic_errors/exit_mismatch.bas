10 WHILE 1
20 EXIT FOR
30 WEND
40 END
