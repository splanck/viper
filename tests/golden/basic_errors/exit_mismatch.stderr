B1011.*EXIT FOR does not match innermost loop \(WHILE\)
